kafka/types.go

```diff
@@ -3,8 +3,10 @@ package kafka
 import (
 	"encoding/json"
 
+	cloudEventSchema "github.com/RedHatInsights/event-schemas-go/apps/exportservice/v1"
 	"github.com/confluentinc/confluent-kafka-go/kafka"
 	"github.com/google/uuid"
+	"gorm.io/datatypes"
```

Comment:
It seems unfortunate that `gorm` is being pulled into the kafka area of the code.  In the future, we might want to consider making the kafka code more generic so that its easier to reuse.  Eventually we might run into a situation where we have different data types (cloud-events, inventory messages, etc) that are getting sent over kafka.  We could modify the kafka interface to take a map of headers (generic map[string]string) and a byte array.  This way the kafka area of the code doesn't know the specifics of the messages...it just knows that it needs to take a couple of generic data types and pass them to kafka.  

Just something to think about in the future.  Its nothing that needs to be done now.  :)

Reply:
Updated with https://github.com/RedHatInsights/export-service-go/pull/90/commits/0dc58a60bbb8ba5867c016eca8871637f777e8b4